lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -767,10 +803,12 @@ def check_watched_item(self, job_state):
                                 pass
                         else:
                             pass
-                    else:
+                    elif self.__check_job_pod_running(job_state):
                         log.debug("Job set to running...")
                         job_state.running = True
                         job_state.job_wrapper.change_state(model.Job.states.RUNNING)
+                    else:
+                        pass
```

Comment:
I think a debug statement would be helpful here. Maybe something like this?:
```suggestion
                    else:
                        log.debug(f"Job id: {job_state.job_id} with k8s id: {job.name} scheduled and is waiting to start...")
```

Reply:
I think info is the right level for that, it's a useful breadcrumb in the context of errors.